voice/voice-ai/x/OpenAIService/OpenAIStreamService.swift

```diff
@@ -34,7 +34,11 @@ class OpenAIStreamService: NSObject, URLSessionDataDelegate {
         self.init(networkService: nil, completion: completion, temperature: temperature)
     }
 
-    init(networkService: NetworkService?, completion: @escaping (String?, Error?) -> Void, temperature: Double = 0.7) {
+    public func setNetworkService(urlSession: URLSession) {
+        self.networkService = urlSession
+    }
+
+    init(networkService: URLSession?, completion: @escaping (String?, Error?) -> Void, temperature: Double = 0.7) {
```

Comment:
Why are there two URLSession variables? One is already lazily initialized in class member

Reply:
I see, this one is only used in tests